src/sass/_common.scss

```
@@ -12,5 +12,4 @@ html {
 
 body {
   font-family: 'Poppins', 'Roboto', sans-serif;
```

```suggestion
body {
  font-family: 'Poppins', 'Aria;', sans-serif;
```

Why don't you sue Arial font? It seems better to me.

---

@volodymyr-ok sry missclicked